unicorecmsebola/templates/search_box.pt

```diff
@@ -0,0 +1,11 @@
+<div xmlns:metal="http://xml.zope.org/namespaces/metal" 
+	 metal:define-macro="search_box"
+	 xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+     i18n:domain="unicorecmsebola">	
+	<div class="search-box">
+		<form action='/search/' method='GET' class='form-search'>
+		    <input id="q" name='q' type="text" hint='enter search here..' i18n:attributes="hint">
```

Comment:
@miltontony Yeah, `placeholder` is the standard. The placeholder text should be an example, though, and I can't think of something generic that would be useful. I say rather just ditch hint.